extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,308 @@
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/binary-poll-raw.png" layout="intrinsic" width="400" height="230">
+
+```html
+<amp-story-interactive-binary-poll
+  id="pizza-binary-poll"
+  endpoint="https://endpoint.com/v1/interactives"
+  prompt-text="Like Pizza?"
+  option-1-text="Yes"
+  option-1-confetti="🍕"
+  option-2-text="No"
+  option-2-confetti="🤢"
+>
+</amp-story-interactive-binary-poll>
+```
+
+### amp-story-interactive-poll
+
+The `amp-story-interactive-poll` element provides a voting experience with 2-4 options displayed vertically, where all options are valid. When selected, each option displays the total percentage of votes.
+
+Display different categories based on user poll answers by pairing `amp-story-interactive-poll` with `amp-story-interactive-results`. Add a prompt for extra context.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/poll-raw.png" layout="intrinsic" width="400" height="450">
+
+[sourcecode:html]
+<amp-story-interactive-poll
+    id="season-poll"
+    theme="dark"
+    endpoint="https://endpoint.com/v1/interactives"
+    prompt-text="Pick a season"
+    option-1-text="Spring" option-1-confetti="🌼"
+    option-2-text="Summer" option-2-confetti="☀️"
+    option-3-text="Fall" option-3-confetti="🍁"
+    option-4-text="Winter" option-4-confetti="☃️">
+</amp-story-interactive-poll>
+[/sourcecode]
+
+### amp-story-interactive-quiz
+
+The `amp-story-interactive-quiz` element provides a guessing experience with 2-4 options, one of which is correct. It displays the voting percentages after the user makes a selection. The user selection is green if correct and red if incorrect.
+
+Display different categories based on percentage of correct user answers by pairing `amp-story-interactive-quiz` with `amp-story-interactive-results`. Add a prompt for extra context.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/quiz-raw.png" layout="intrinsic" width="400" height="450">
+
+[sourcecode:html]
+<amp-story-interactive-quiz
+    id="arts-quiz"
+    endpoint="https://endpoint.com/v1/interactives"
+    prompt-text='Who was the artist that created the famous painting "The Last Supper"?'
+    option-1-text="Michelangelo"
+    option-2-text="Leonardo da Vinci" option-2-correct option-2-confetti="🎨"
+    option-3-text="Rahael"
+    option-4-text="Sandro Boticelli">
+</amp-story-interactive-quiz>
+[/sourcecode]
+
+### amp-story-interactive-results
+
+The `amp-story-interactive-result` element displays a customized state defined by the user's selection from previous polls or quizzes. This element requires use of a poll or quiz from previous pages to calculate an answer-based state. Each result state uses a category that may include an image, title and description to display to the user.
```

Comment:
Publishers have to use exclusively polls or quizzes for it to work, right? Can we mention that as well?

Reply:
Yes! Currently we have "This element requires use of a poll or quiz from previous pages to calculate an answer-based state.". Anything else we are missing?